scripts/set-netlify-deployment-status.js

```diff
@@ -31,6 +32,10 @@ const run = async () => {
     return;
   }
 
+  const contextName = !DEPLOY_CONTEXT
```

Comment:
Can be named `context` to have an object [value shorthand](https://alligator.io/js/object-property-shorthand-es6/) later like 

```
payload = {
        state: 'success',
        target_url: deploymentResult.deploy_url,
        description: 'Preview ready!',
        context,
      };
```

Feel free also to break out of `drone/netlify` naming as it was quite a brisk initial one around the preview feature.

Maybe something like `drone/twig-js` and `drone/twig-php` will be more descriptive in your case to compare 2 previews related to rendering options.